helm-chart/mezod/templates/statefulset.yaml

```diff
@@ -111,6 +119,11 @@ spec:
             - name: custom-configs
               mountPath: /config/
             {{- end }}
+            {{- if .Values.secrets.node_key }}
+            - name: node-key
+              mountPath: "/var/mezod/config/node_key.json"
```

Comment:
`/var/mezod/` can be set based on environment variable `MEZOD_HOME`

Reply:
Addressed!